### PR TITLE
Fixes issue with the file dialog showing only folders and not files

### PR DESCRIPTION
**Pull request by jjvn84**
Fixes issue with the file dialog showing only folders and not files when the <input> element has no "accept" attribute. If the "accept" attribute is None, it changes it to ".*", which instructs the file dialog to show files with all possible extensions.
Also changes the filter's name in the file dialog to show a list of acceptable extensions.

Partially fixes [https://github.com/DioxusLabs/dioxus/issues/3439](https://github.com/DioxusLabs/dioxus/issues/3439)

### CODE REVIEW

File: packages/desktop/src/file_upload.rs [hunk 1]
```diff
@@ -79,7 +79,7 @@ impl FileDialogRequest {
             let filters: Vec<_> = request
                 .accept
                 .as_deref()
-                .unwrap_or_default()
+                .unwrap_or(".*")
                 .split(',')
```
Comment (TheLostLambda) [inline packages/desktop/src/file_upload.rs:83]: It'll need testing, but we might need to trim whitespace from the strings after this? It looks like you can have spaces after the `,` in `accept` (see examples in https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/accept#limiting_accepted_file_types — `accept=".png, .jpg, .jpeg"`).

When I added spaces, it didn't seem to work for me, but I think it should!

Comment (jjvn84) [reply packages/desktop/src/file_upload.rs:83]: You are right, I'll add a ```trim()``` to the ```filter_map``` below those lines

**Review by jkelleyrtp — APPROVED** [review]

**Comment by jkelleyrtp** [comment]
Thank you! We still need to fix the other half of the original issue as well as the associated windows bug.
